instrumentation/runtime/example/main.go

```diff
@@ -27,13 +27,17 @@ var res = resource.NewWithAttributes(
 )
 
 func main() {
-	exp, err := stdoutmetric.New()
+	exp, err := stdoutmetric.New(stdoutmetric.WithPrettyPrint())
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	// Register the exporter with an SDK via a periodic reader.
-	read := metric.NewPeriodicReader(exp, metric.WithInterval(1*time.Second))
+	read := metric.NewPeriodicReader(
+		exp,
+		metric.WithInterval(1*time.Second),
+		metric.WithProducer(runtime.NewProducer()),
+	)
```

Comment:
Given we add a dependency to the SDK, I think we can refactor this to a "testable example" in `go.opentelemetry.io/contrib/instrumentation/runtime` to make its docs better.

Reply:
Do you think it should use the stdoutmetric exporter?

Reply:
We could do something like https://github.com/open-telemetry/opentelemetry-go/blob/4d29b35a1b6f78a85a91d8a8657ae521f57fb6b0/sdk/metric/example_test.go#L39

Reply:
done